CHANGES.rst

```diff
@@ -1,7 +1,7 @@
 Changes
 =======
 
-Version 0.7.1 (UNRELEASED)
+Version 0.7.1 (2020-11-10)
 --------------------------
 
 - Fixes problem on GitLab integration preventing the synchronisation of projects with existing webhooks.
```

Comment:
Can you please complete also WRT ping-related, restart-related, and perhaps also celery-related fixes?

```console
$ git log 0.7.0..upstream/pr/316 --format=oneline
f870cfa43f54d7b03581440341cf43754109d2c5 (upstream/pr/316) release: 0.7.1
fa209961becbd43fcb2a4b4dfaf11b98ffba7c5c (HEAD -> maint-0.7, upstream/pr/315, upstream/maint-0.7, origin/maint-0.7) gitlab: fix fetching projects with >
fc393cdf2d96b6bdf97c26cc5674033966478b83 (upstream/pr/312) rest: return reana_server url
32ab3e24ca46e623f98ece2322dacfd5d5a2b4fd (upstream/pr/294) rest: allow restart with different languages in same workspace
0d92dfb3490a4862ea05e5c50458d20dcb91550f (upstream/pr/310) setup: pin celery version <5
247e3a3adfcf298312fd7e85efc60aaf1bc2437d (upstream/pr/309) ci: added github actions workflow
```
```